tests/PHPSemVerChecker/Analyzer/ClassMethodAnalyzerTest.php

```diff
@@ -708,4 +708,54 @@ public function providerImplementationChanged()
 			['trait', 'private', 'V054'],
 		];
 	}
+
+	/**
+	 * @dataProvider providerCaseChanged
+	 */
+	public function testClassMethodCaseChangeChanged($context, $visibility, $code)
```

Comment:
The method name shouldn't be testClassMethodCaseChangeChanged but testClassMethodNameCaseChanged.